08/kth-missing-positive-number.checkpoint.md

```diff
@@ -208,6 +208,106 @@ Code solution to a subproblem
 
 <!-- ======================= END CHALLENGE ======================= -->
 
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: code-snippet
+* language: python3.6
+* id: 5a380194-96e2-4b87-9aa7-2fe407448e1f
+* title: Coding Solution
+* points: 3
+* topics: python, pse, python-lists
+
+##### !question
+
+Write the code to solve the problem here.  You can use the tests to guide your solution.
+
+##### !end-question
+
+##### !placeholder
+
+```python
+def kth_missing_positive_number(numbers, k):
+    '''
+    INPUT: List of numbers in increating order & a positive integer k
+    OUTPUT: The kth missing number
+    '''
+
+    pass
+```
+
+##### !end-placeholder
+
+##### !tests
+
+```py
+import unittest
+import main as p
+import numpy as np
+
+class TestPython1(unittest.TestCase):
+  def test_example_one(self):
```

Comment:
Woops, forgot to add these comments in my last review, sorry >_<

More descriptive test names that describe the situation may help. Also, referencing the tested function name would help.

Gonna make similar suggestions for each test name

Leaving it without a situation since it's our prototypical case

```suggestion
  def test_kth_missing_positive_number(self):
```